README.md

```diff
@@ -59,6 +59,21 @@ run(main, {
 })
 ```
 
+By default `@cycle/run` uses `xstream`. If you want to use another streaming library in your code simply import them and use their `run` method instead.
+
+For RxJS:
+
+```js
+import { run } from '@cycle/rxjs-run';
```

Comment:
Do you think we should add import Rx from ... etc on those examples as well for clarity?